tests/test_stereoshift_dialog.py

```diff
@@ -23,15 +25,31 @@
         ),
         (
             [
+                np.array([0, 0]),
+                np.array([0, 0]),
+                np.array([0, 2]),
+                np.array([2, 0]),
+                np.array([0, 0]),
+                np.array([1, 0]),
+            ],
+            sqrt(8),
+            1.0,
+            1 / sqrt(8),
+            1 / sqrt(8) * CD,
+        ),
+        (
+            [
+                np.array([0, 1]),
+                np.array([1, 0]),
                 np.array([0, 0]),
                 np.array([2, 2]),
                 np.array([-1, 0]),
                 np.array([0, -1]),
             ],
-            sqrt(8),
-            sqrt(2),
-            0.5,
-            0.5 * CD,
+            sqrt(10),  # sqrt(8)
+            sqrt(0),  # sqrt(2)
+            0.0,  # 0.5
+            0.0 * CD,  # 0.5*CD
```

Comment:
```suggestion
            sqrt(10),
            sqrt(0),
            0.0,
            0.0 * CD,
```
A lot of magic numbers here. And why `CD` rather than `CHAMBER_DEPTH` (which is a nice meaningful name)?